### DISCUSSION TITLE
How to change RCon password

### DISCUSSION THREAD

**Discussion by DevDroid1** [itzg/docker-minecraft-server#3049] [Q&A]
so i got my paper mc server working but I don't have op to use some commands. i want to setup RCon but it keeps asking for my password
I copied the password from my server properties file but it still says its incorrect
can someone tell me how to set my own password
(if you try to change it in server properties it get regenerated when I restart the container)

I am using portainer on a sysnology NAS

**Comment by itzg** [comment]
Please read https://docker-minecraft-server.readthedocs.io/en/latest/configuration/server-properties/#rcon
